<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RenameTextureDialog</class>
 <widget class="QDialog" name="RenameTextureDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>238</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>367</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newtexturename" >
   <property name="geometry" >
    <rect>
     <x>82</x>
     <y>10</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Texture Name&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Search Texture</string>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>RenameTextureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RenameTextureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
